chapter_12_organising_test_files.asciidoc

```diff
@@ -561,9 +568,10 @@ saves you from having to use `def..():` and an indented block:
 5
 ----
 
-In our case, we're using it to transform a bit of code that would otherwise be
-executed immediately into a function that we can pass as an argument,
+In our case, we're using it to transform a bit of code - that would otherwise be
+executed immediately - into a function that we can pass as an argument,
```

Comment:
```suggestion
In our case, we're using it to transform a bit of code--that would otherwise be
executed immediately--into a function that we can pass as an argument,
```